### PR TITLE
Merge updates to master

### PR DESCRIPTION
**Pull request by m1cr0man**
Separated out the mailman stuff. Includes the other mail stuff for a fully functioning postfix + dovecot setup.

### CODE REVIEW

**Comment by m1cr0man** [comment]
Please note: Use `nixos-rebuild boot` and reboot hardcase since I changed the postgres data path

File: services/httpd/vhosts.nix
Comment (butlerx) [inline services/httpd/vhosts.nix:32]: so I think it'd be a good idea if we alphabetise this list. https://github.com/redbrick/nix-configs/blob/blog/services/httpd/vhosts.nix <- was done in the testing blog so the only difference is the lack of blog

File: services/httpd/vhosts.nix
Comment (butlerx) [inline services/httpd/vhosts.nix]: this was deleted from main

File: services/httpd/vhosts.nix
Comment (butlerx) [inline services/httpd/vhosts.nix]: delete?

**Review by butlerx — CHANGES_REQUESTED** [review]

**Comment by butlerx** [comment]
let's send out a maintenance notice for the reboot

File: services/httpd/vhosts.nix
Comment (m1cr0man) [reply services/httpd/vhosts.nix:32]: Yeah fair enough. Shall we not wait until the blog pr is merged then?

**Comment by m1cr0man** [comment]
I actually deleted 3 vhosts. I'll reintroduce that patch

File: services/httpd/vhosts.nix
Comment (butlerx) [reply services/httpd/vhosts.nix:32]: sure it can wait till #38

**Review by butlerx — APPROVED** [review]

**Pull request merged**
